codec-http/src/test/java/io/netty/handler/codec/http/multipart/HttpPostRequestDecoderTest.java

```diff
@@ -1002,4 +1003,26 @@ public void testDecodeMultipartRequest() {
             assertTrue(req.release());
         }
     }
+
+    /**
+     * when diskFilename contain "\" create temp file error
+     */
+    @Test
+    void testHttpPostStandardRequestDecoderToDiskNameContainingUnauthorizedChar() throws UnsupportedEncodingException {
+        StringBuffer sb = new StringBuffer();
+        byte[] bodyBytes ="aaaa/bbbb=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa".getBytes();
+        ByteBuf content = Unpooled.directBuffer(bodyBytes.length);
+        content.writeBytes(bodyBytes);
+
+        FullHttpRequest req = new DefaultFullHttpRequest(HttpVersion.HTTP_1_1, HttpMethod.POST, "/", content);
+        try {
+            HttpPostStandardRequestDecoder decoder = new HttpPostStandardRequestDecoder(new DefaultHttpDataFactory(true), req);
+            decoder.offer(req);
+        } catch (HttpPostRequestDecoder.ErrorDataDecoderException e) {
+            assertEquals("java.lang.IllegalArgumentException: Invalid prefix or suffix", e.getMessage());
```

Comment:
@EmiyaXzero I don't get why you expect such an exception? Shall it be the reverse, meaning if there is an exception, this is an issue?

Reply:
Thanks for the reminder,I have modified this issue